bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/api/FreeboxOsIconProvider.java

```diff
@@ -51,6 +52,7 @@
 public class FreeboxOsIconProvider extends AbstractResourceIconProvider {
 
     private final Logger logger = LoggerFactory.getLogger(FreeboxOsIconProvider.class);
+    private static final int REQUEST_TIMEOUT_MS = 8000;
```

Comment:
By curiosity, why this value ?

@clinique : is a reasonable value in your opinion ?

Reply:
Because of the 'business rule' i mentioned in the startpost. Elsewhere this binding has timeouts set to 8 seconds.
https://github.com/openhab/openhab-addons/blob/c5739eccc978f520473750871074d1e94a7579d5/bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/api/ApiHandler.java#L69